adapters/aws_redshift.go

```diff
@@ -25,6 +25,11 @@ type AwsRedshift struct {
 	s3Config        *S3Config
 }
 
+type RedshiftConfig struct {
```

Comment:
do we need RedshiftConfig here?

Reply:
I use it as a DTO and use RedshiftConfig at another client that sends configuration for validation